flake.nix

```diff
@@ -111,6 +114,7 @@
               RUSTFLAGS = pkgs.lib.optionalString pkgs.stdenv.isLinux "-C link-arg=-fuse-ld=mold";
               NIX_JJ_GIT_HASH = self.rev or "";
               CARGO_INCREMENTAL = "0";
+              TEST_GIT_EXECUTABLE_PATH = pkgs.lib.getExe pkgs.git;
```

Comment:
nit: if we can just rely on the `$PATH`, it might be nice? (I don't know nix, so I have no idea whether that makes sense.)